go_module/Dockerfile

```diff
@@ -28,6 +28,7 @@ WORKDIR $GOPATH/src/mypackage/myapp/
 COPY go.mod .
 
 ENV GO111MODULE=on
+RUN go mod tidy
```

Comment:
please add this to other Dockerfile using mod

Reply:
done